system/jhtree/jhtree.cpp

```diff
@@ -652,7 +652,8 @@ class CNodeMRUCache : public CMRUCacheOf<CKeyIdAndPos, CJHTreeNode, CNodeMapping
             CNodeMapping &mapping = iter->query();
             const CKeyIdAndPos &key = mapping.queryFindValue();
             const CJHTreeNode &node = mapping.queryElement();
-            cacheInfo.noteWarm(key.keyId, key.pos, node.getNodeSize(), node.getNodeType());
+            if (node.isReady())
```

Comment:
Gavin's new cache scheme can leave nodes in the cache that are not fully set up.